Makefile

```diff
@@ -27,85 +30,26 @@ pip-check:
 # Build and run production docker container
 
 build-prod:
-	docker build --file=Dockerfile . --tag=openslides-backend
+	docker build . --tag=openslides-backend
 
-run-prod:
+run-prod: | build-prod
 	docker run --interactive --tty \
 	--publish 9002:9002 --publish 9003:9003 --rm openslides-backend
 
 
-# Build and run development and testing docker container
-
-build-dev:
-	docker build --file=Dockerfile-dev . --tag openslides-backend-dev
-
-mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
-
-action_args=--interactive \
-	--volume=$(dir $(mkfile_path))openslides_backend:/srv/code/openslides_backend \
-	--volume=$(dir $(mkfile_path))tests:/srv/code/tests \
-	--network=host --rm openslides-backend-dev
-dev_args=--tty $(action_args)
-
-run-dev: | build-dev
-	docker run $(dev_args)
-
-run-dev-interactive run-bash: | build-dev
-	docker run $(dev_args) sh
-
-run-cleanup: | build-dev
-	docker run $(dev_args) sh cleanup.sh
-
-run-unit-tests: | build-dev
-	docker run $(dev_args) pytest tests/unit/
-
-
-# testing
-# -T is important for GH actions: There is no TTY
-
-DC=docker-compose -f docker-compose.dev.yml
-
-build-tests: | build-dev
-	$(DC) build
-
-start-test-setup: | build-tests
-	$(DC) up -d
-	$(DC) exec -T backend ./wait.sh writer 9011
-	$(DC) exec -T backend ./wait.sh reader 9010
-
-start-test-setup-verbose: | build-tests
-	$(DC) up
-	$(DC) exec -T backend ./wait.sh writer 9011
-	$(DC) exec -T backend ./wait.sh reader 9010
-
-run-tests-interactive: | start-test-setup
-	$(DC) exec backend sh
-
-run-tests: | start-test-setup
-	$(DC) exec backend pytest
-	$(DC) down --volumes
-
-run-tests-cov: | start-test-setup
-	$(DC) exec backend pytest --cov
-	$(DC) down --volumes
-
-ci-run-tests-cov: | start-test-setup
-	$(DC) exec -T backend pytest --cov
-	$(DC) down --volumes
-
-stop-tests:
-	$(DC) down --volumes
+# Build and run development docker container setup with docker compose
 
-# other github actions commands (no tty available)
+start-dev:
+	docker-compose -f dev/docker-compose.dev.yml up --build --detach
 
-ci-black-check: | build-dev
-	docker run $(action_args) black --check --diff openslides_backend/ tests/
+stop-dev:
+	docker-compose -f dev/docker-compose.dev.yml down --volumes
 
-ci-isort-check: | build-dev
-	docker run $(action_args) isort --check-only --diff openslides_backend/ tests/
+start-dev-interactive:
+	docker-compose -f dev/docker-compose.dev.yml up  --build
 
-ci-flake8-check: | build-dev
-	docker run $(action_args) flake8 openslides_backend/ tests/
+run-dev run-bash:
```

Comment:
There should be a dependency to start the containers. I cannot do `run-dev` stand alone anymore.

Reply:
I understood that you do not run stand alone dev container because you always use docker compose dev setup.

Reply:
> because you always use docker compose dev setup.

Yes, but if I do `make run-bash`, the setup isn't started and I get an error.

Reply:
Ok I got it. But then it is more difficult to start the docker compose setup in an extra terminal and run everything with an extra command. But I can change this.